modules/ROOT/pages/upgrade-service-mesh.adoc

```diff
@@ -0,0 +1,60 @@
+= Upgrade Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, upgrading
+
+
+Anypoint Service Mesh v1.1 includes improvements, such as the removal of the Istio Mixer requirement. Because of this change, upgrading from Anypoint Service Mesh v1.0 to v1.1 introduces some downtime due to the new architecture.
```

Comment:
```suggestion
Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as stopping using Istio Mixer. Because of this change, upgrading from Anypoint Service Mesh v1.0 to v1.1 introduces downtime during the c.
```

Reply:
Done! Thanks!